src/main/java/org/opensearch/security/dlic/rest/api/FlushCacheApiAction.java

```diff
@@ -39,9 +39,21 @@ public class FlushCacheApiAction extends AbstractApiAction {
     private static final List<Route> routes = addRoutesPrefix(
         ImmutableList.of(
             new Route(Method.DELETE, "/cache"),
-            new Route(Method.GET, "/cache"),
-            new Route(Method.PUT, "/cache"),
-            new Route(Method.POST, "/cache")
+            new DeprecatedRoute(
+                Method.GET,
+                "/cache",
+                "GET is not supported for /cache endpoint and will be removed in the next major version."
```

Comment:
```suggestion
                "GET is not supported for /cache endpoint and will be removed in the next major version. Use DELETE instead."
```

Reply:
I don't think we can necessarily assume that they want to DELETE the cache - all we should say is we don't support GET/POST/PUT - wdyt?

Reply:
DELETE is the HTTP Verb, so its not deleting the cache, but instructing the nodes to refresh their caches from the security index.

A caller of this API using GET/POST/PUT will already get a not implemented response. I was thinking that the message could be more informative and tell the user the correct http verb to use.

Reply:
If they are using this API then we can assume the intent is to flush the cache.

Reply:
Sure makes sense. Added that line to the end of all other routes as well in: [8c99aa0](https://github.com/opensearch-project/security/pull/4776/commits/8c99aa0c07286e28e6628e3d0e814235691a76a6)